backend/test/setup.js

```diff
@@ -55,8 +58,17 @@ before(async function () {
 if (isWatchMode) {
   process.once('exit', () => process.exit(0))
 } else {
-  // Shutdown ganache etc if we're not in watch mode and tests are finished.
+  // Shutdown services and clean up if we're not in watch mode and tests are finished.
   after(async function () {
     await shutdownServices()
+
+    console.log(`Cleaning up test data in ${TEST_TMP_DIR}...`)
+    // Note: fs.rmdirSync(TEST_TMP_DIR, { recursive: true }) is still experimental and does not seem to work.
+    await new Promise((resolve, reject) => {
+      execFile('rm', ['-rf', TEST_TMP_DIR], (err) => {
```

Comment:
Can we be a bit more defensive about `rm -rf`?  Also, and env var to maybe disable this could be useful.  There are times where I want to inspect the dir structure.

Reply:
yes good point. I'll add a check to make sure TEST_TMP_DIR starts with /tmp
Also will add an option or env var to leave the data.